arcgis-ios-sdk-samples/Edit data/Edit and sync features/EditAndSyncFeaturesViewController.swift

```diff
@@ -85,15 +85,16 @@ class EditAndSyncFeaturesViewController: UIViewController {
             if let error = error {
                 self.presentAlert(error: error)
             } else {
-                if let featureServiceInfo = self.geodatabaseSyncTask?.featureServiceInfo,
-                    let map = self.mapView.map {
-                    for index in featureServiceInfo.layerInfos.indices.reversed() {
-                        let layerInfo = featureServiceInfo.layerInfos[index]
-                        // For each layer in the serice, add a layer to the map.
-                        if let layerURL = self.featureServiceURL?.appendingPathComponent(String(index)) {
-                            let featureTable = AGSServiceFeatureTable(url: layerURL)
+                guard let featureServiceInfo = self.geodatabaseSyncTask?.featureServiceInfo, let map = self.mapView.map else { return }
+                for index in featureServiceInfo.layerInfos.indices {
+                    // For each layer in the serice, add a layer to the map.
+                    guard let layerURL = self.featureServiceURL?.appendingPathComponent(String(index)) else { return }
+                    let featureTable = AGSServiceFeatureTable(url: layerURL)
+                    featureTable.load { error in
+                        if let error = error {
+                            self.presentAlert(error: error)
+                        } else if featureTable.geometryType == .point {
                             let featureLayer = AGSFeatureLayer(featureTable: featureTable)
-                            featureLayer.name = layerInfo.name
                             map.operationalLayers.add(featureLayer)
                         }
                     }
```

Comment:
It might not need to explicitly call load here, but I'm not sure if the `geometryType` depends on loading status. Wait for Phil's comments.

Reply:
That's a good question. @vquach2404 Does the sample function as expected without explicitly loading each table?

Reply:
I played around with it, trying to load only the point layer. It seems like it's unable to determine the `geometryType` of the layer unless it's loaded.